packages/components/forms/src/base-checkbox/BaseCheckboxGroup.tsx

```diff
@@ -4,6 +4,10 @@ import type { CommonProps } from '@contentful/f36-core';
 import { BaseCheckboxGroupContext } from './BaseCheckboxGroupContext';
 
 export interface BaseCheckboxGroupProps extends CommonProps {
+  /**
+   * Handler that will be triggered when any checkbox inside the group loses focus
+   */
+  onBlur?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
Would also need to add to the import on line 1
```suggestion
  onBlur?: FocusEventHandler<HTMLInputElement>;
```

Reply:
🤦